java/cuvs-java/src/main/java/com/nvidia/cuvs/SearchResults.java

```diff
@@ -18,9 +18,24 @@
 
 import java.util.List;
 import java.util.Map;
+import java.util.function.LongToIntFunction;
 
 public interface SearchResults {
 
+    /**
+     * The default identity function mapping neighbours IDs to user-defined IDs
+     */
+    LongToIntFunction IDENTITY_MAPPING = l -> (int) l;
```

Comment:
Should this be a `static final`?

Reply:
SearchResults is an interface, so `static final` is redundant. But if you think it makes it clearer, I can add it